### PR TITLE
Add ability to customise Label font size and offset

### PR DESCRIPTION
**Pull request by SerhiiTsybulskyi**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[x] No
```

### CODE REVIEW

File: docs/Theme.mdx [hunk 1]
```diff
@@ -66,6 +66,8 @@ export interface Theme {
     label?: {
       stroke?: ColorRepresentation;
       color: ColorRepresentation;
+      fontSize?: number;
+      yOffset?: number;
```
Comment (amcdnl) [inline docs/Theme.mdx:70]: I would change this to `offset: [number, number, number]` = x/y/z

File: src/themes/theme.ts [hunk 2]
```diff
@@ -56,6 +56,8 @@ export interface Theme {
     label?: {
       stroke?: ColorRepresentation;
       color: ColorRepresentation;
+      fontSize?: number;
+      yOffset?: number;
```
Comment (amcdnl) [inline src/themes/theme.ts:60]: Need to add docs to the props

**Review by amcdnl — CHANGES_REQUESTED** [review]
few minor nits

File: src/themes/theme.ts (hunk 2 above)
Comment (SerhiiTsybulskyi) [reply src/themes/theme.ts:60]: @amcdnl What do you mean? Comment with description above or something else?

Comment (amcdnl) [reply src/themes/theme.ts]: Yes

File: src/themes/theme.ts [hunk 3]
```diff
@@ -56,6 +56,10 @@ export interface Theme {
     label?: {
       stroke?: ColorRepresentation;
       color: ColorRepresentation;
+      // Size of the cluster label
+      fontSize?: number;
+      // Offset of the cluster label relative to the default position
```
Comment (amcdnl) [inline src/themes/theme.ts:61]:
```
/**
 * Offset of the cluster label relative to the default
 */
```

**Review by amcdnl — APPROVED** [review]

**Pull request merged**